lib/swaps/Swaps.ts

```diff
@@ -403,7 +406,12 @@ class Swaps extends EventEmitter {
 
     const rHash = requestPacket.body!.rHash;
     if (this.usedHashes.has(rHash)) {
-      await this.sendErrorToPeer(peer, requestPacket.body!.rHash, SwapFailureReason.PaymentHashReuse, undefined, requestPacket.header.id);
+      await this.sendErrorToPeer({
+        peer,
+        rHash: requestPacket.body!.rHash,
```

Comment:
You can just write this as `rHash,` right? Since we declare it above on line 407.